.github/workflows/main_sleuth-modified.yml

```diff
@@ -0,0 +1,63 @@
+# Docs for the Azure Web Apps Deploy action: https://github.com/Azure/webapps-deploy
+# More GitHub Actions for Azure: https://github.com/Azure/actions
+# More info on Python, GitHub Actions, and Azure App Service: https://aka.ms/python-webapps-actions
+
+name: Build and deploy Python app to Azure Web App - sleuth-modified
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:

```

Comment:
This deployment should not be part of the PR

Reply:
ah sorry I need to deploy from the main branch of the forked repo which I did after I made this PR. I'll close this PR and make another one from a different branch that doesn't have all these extra commits.